cockatrice/src/chatview.h

```diff
@@ -44,6 +44,7 @@ private slots:
     ChatView(const TabSupervisor *_tabSupervisor, TabGame *_game, bool _showTimestamps, QWidget *parent = 0);
     void retranslateUi();
     void appendHtml(const QString &html);
```

Comment:
`appendHtml` is called from messagelogeidget.cpp, which extends chatview. That is the only place I see the call being made from. I could make both `appendHtml` and `appendHtmlServerMessage` protected instead?